app/views/categories/new.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="categories">
+    <div class="cat-header new-category">
+        <div class="top">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+            <h2>NEW CATEGORY</h2>
+            <%= image_tag("search-icon.png", alt: "search icon", class: "search-icon") %>
+        </div>
+        
+        <div class="category-bottom-header">
+            <%= link_to "", categories_path, class: "fas fa-arrow-left" %>
+
+            <div class="middle">
+                <% if user_signed_in? %>
+                    <form action="<%= destroy_user_session_path %>" method="post">
+                    <input type="hidden" name="_method" value="delete">
+                    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+                    <button type="submit" class="btn btn-danger">Logout</button>
+                    </form>
+                <% end %>
+            </div>
+        </div>
+    </div>
+
```

Comment:
- [ ] You still have the styling issue from the previous reviewer. Kindly fix it.

  ![image](https://github.com/anyars-encarta/Rails-Capstone-Budget/assets/54848363/b4035026-bba6-4f16-81f1-13e0ed1a6218)